deploy.py

```diff
@@ -866,10 +868,18 @@ def run(self) -> int:
               "configured yet.")
       return 0
 
-    # Perform checks if OpenSK was flashed.
-    if self.args.application != "ctap2":
+    if "vendor_hid" in self.args.features:
+      # vendor_hid as a work in progress and is not compatible with configure
+      # mode.
       return 0
 
+    # Perform checks if OpenSK was flashed.
+    if self.args.application == "ctap2" or self.args.configure:
```

Comment:
Do we still need the configure flag when configuration is guarded by this?
`if "vendor_hid" in self.args.features:`

I think giving users the choice is not bad, but its default should then be `true`. If we introduce it, we should mention it in `docs/customization.md` where users read about the privacy trade-offs of configuration.

Reply:
This PR was stacked on top of #471.
I've separated these, as this check means these features are unrelated.